test/integration/coscheduling_test.go

```diff
@@ -118,118 +118,127 @@ func TestCoschedulingPlugin(t *testing.T) {
 		{
 			name: "equal priority, sequentially pg1 meet min and pg2 not meet min",
 			pods: []podInfo{
-				{podName: "t1-p1-1", podGroupName: "pg1", minAvailable: "3", priority: midPriority},
-				{podName: "t1-p1-2", podGroupName: "pg1", minAvailable: "3", priority: midPriority},
-				{podName: "t1-p1-3", podGroupName: "pg1", minAvailable: "3", priority: midPriority},
-				{podName: "t1-p2-1", podGroupName: "pg2", minAvailable: "4", priority: midPriority},
-				{podName: "t1-p2-2", podGroupName: "pg2", minAvailable: "4", priority: midPriority},
-				{podName: "t1-p2-3", podGroupName: "pg2", minAvailable: "4", priority: midPriority},
-				{podName: "t1-p2-4", podGroupName: "pg2", minAvailable: "4", priority: midPriority}},
+				{podName: "t1-p1-1", podGroupName: "pg1-1", minAvailable: "3", priority: midPriority, memReq: 50},
```

Comment:
the new `podGroupName` is to distinguish each sub-test, otherwise, this test would fail due to PodGroup having different PG names.

Reply:
Making the memory request to 50 is to tolerate the case that both "t1-p1-3" and "t1-p2-4" co-exist in the waiting map.